packages/appeals-service-api/src/controllers/appeals.js

```diff
@@ -0,0 +1,27 @@
+const { getAppealsByEmail } = require('../services/appeal.service');
+const logger = require('../lib/logger');
+const ApiError = require('#errors/apiError');
+
+const getUserAppealsByEmail = async (req, res) => {
```

Comment:
This is to get all the appeals for a given user email. I used it to check a user has no associated appeals before enrolling them.